app/src/main/AndroidManifest.xml

```diff
@@ -413,6 +413,11 @@
             android:authorities="com.getpebble.android.provider"
             android:exported="true" />
 
+        <provider
+            android:name=".contentprovider.HRContentProvider"
+            android:authorities="com.gadgetbridge.heartrate.provider"
```

Comment:
gadgetbridge.com is not our domain, gadgetbridge.org is. But why do you explicitly not use the nodomain.freeyourgadget.org name?

Reply:
Indeed, I wanted to follow the name of the java packages, however on a second thought this should then have been **nodomain.freeyourgadget.gadgetbridge**. I'll change it to org.gadgetbridge. Because I a am tempted to keep the provider more general i'll go with

`org.gadgetbridge.realtimesamples.provider`

Feel free to oppose